spec/features/request_dashboard_spec.rb

```diff
@@ -78,5 +80,39 @@
       click_button 'Decline'
       expect(page).not_to have_button 'Decline'
     end
+
+    scenario 'should display message if no requests have been received' do
+      click_button "Sign out"
+      signin(host3.email,1234)
+      visit('requests')
+      expect(page).to have_content('It seems like you haven\'t received any request yet')
+    end
+  end
+
+  feature 'Guest' do
+    before :each do
+      signin(guest1.email,1234)
+      visit('requests')
+    end
+
+    scenario 'should have a "Requests I\'ve made"' do
+      expect(page.status_code).to eq 200
+      expect(page).to have_content 'Requests I\'ve made'
+    end
+
+    scenario 'should see their own requests' do
+      expect(page).to have_content('BigBen')
+      expect(page).to have_content('Host1')
+      expect(page).to have_content('Status: Pending')
+      expect(page).to have_content('Date requested: 2016-05-12')
+    end
+
+    scenario 'should display message if no requests have been made' do
+      click_button "Sign out"
+      signin(guest2.email,1234)
+      visit('requests')
+      expect(page).to have_content('It seems like you haven\'t made any request yet')
+    end
+
```

Comment:
Missing test to make sure the status of request can change. I.e the user story: I want to see if my request was confirmed